x-pack/plugins/cloud/public/services/index.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, createContext, useContext } from 'react';
+
+interface WithoutChat {
+  enabled: false;
+}
+
+interface WithChat {
+  enabled: true;
+  chatURL: string;
+  identityJWT: string;
```

Comment:
Could `jwt`, `id` and `email` properties be a part of the `user` object?

Reply:
Which `user` object?

Reply:
I meant, if the signature looked something like:
`
enabled: boolean;
chatURL: string;
user : {
   jwt: 
   id:
   email:
 }
`